projects/gateway2/helm/gloo-gateway/templates/gateway/proxy-deployment.yaml

```
@@ -89,15 +89,16 @@ spec:
         - name: http-monitoring
           containerPort: 9091
         {{- end }}
-        exec:
-          command:
-            - wget
-            - -O
-            - /dev/null
-            - 127.0.0.1:19000/ready
-          initialDelaySeconds: 3
-          periodSeconds: 10
-          failureThreshold: 3
+        livenessProbe:
+          exec:
+            command:
+              - wget
+              - -O
+              - /dev/null
+              - 127.0.0.1:19000/ready
+            initialDelaySeconds: 3
+            periodSeconds: 10
+            failureThreshold: 3
```

```suggestion
       {- $gateway.probes.livenessprobe }
```

or something like that

---

Will add that once we add support for custom livenss probes

---

but i dont think we want a default liveness at all

---

The issue description says "I'd like to have proper probes by default". Seems like you and Jesus (issue author) should align on desired behavior here.

---

we should consider how it's done in the [edge proxies](https://github.com/solo-io/gloo/blob/main/install/helm/gloo/templates/7-gateway-proxy-deployment.yaml) and decide if we want similar configurability here